membership_file/tests/tests_export.py

```diff
@@ -31,14 +31,16 @@ def test_accessible_rooms(self):
         """ Tests if accessible rooms are added correctly """
         for row in self.reader:
             if row['id'] == 1:
-                self.assertIn(str(Room.objects.get(id=1)), row["accessible_rooms"])
-                self.assertIn(str(Room.objects.get(id=2)), row["accessible_rooms"])
+                self.assertIn(str(Room.objects.get(id=1)),
+                              row["accessible_rooms"])
+                self.assertIn(str(Room.objects.get(id=2)),
+                              row["accessible_rooms"])
                 break
 
 
 class MembershipFinanceFileExportTest(TestCase):
     """ Tests the structure of the exported membership file """
-    fixtures = ['test_users', 'test_members',]
+    fixtures = ['test_users', 'test_members', ]
```

Comment:
Unrelated, but are all the fixtures used btw?

Reply:
Many fixtures contain references to other fixtures. For instance, `test_members` call users from `test_users`. The same applies for many other fixtures. `test_activity_slots` also use `test_members`, or the fixture for the committees module which uses `test_membes` as well.